src/Relude/Base.hs

```diff
@@ -76,11 +65,7 @@ import Data.Void (Void, absurd, vacuous)
 
 import GHC.Base (String, asTypeOf, maxInt, minInt, ord, seq, ($!))
```

Comment:
GHC module structure is kinda awkward... I feel like `maxInt` and `minInt` also should go to `Relude.Numeric`. This module looks like better place for such things and should be the only place to search for all numeric/integral/fractional things.

Reply:
yep I missed those two!